src/views/preview/presentation.jsx

```diff
@@ -1,332 +1,416 @@
-                                            value={Date.parse(shareDate)}
-                                            day="2-digit"
-                                            month="short"
-                                            year="numeric"
-                                        />
+                                        {shareDate === null ?
+                                            'Unshared' :
+                                            <FormattedDate
+                                                value={Date.parse(shareDate)}
+                                                day="2-digit"
+                                                month="short"
+                                                year="numeric"
+                                            />
+                                        }
                                         {/*  eslint-enable react/jsx-sort-props */}
                                     </div>
-                                )}
-                                {creditInfo && creditInfo.author && creditInfo.id && (
-                                    <FlexRow className="remix-credit">
-                                        <Avatar
-                                            className="remix"
-                                            src={`https://cdn2.scratch.mit.edu/get_image/user/${creditInfo.author.id}_48x48.png`}
-                                        />
-                                        <div className="credit-text">
-                                            Thanks to <a
-                                                href={`/users/${creditInfo.author.username}`}
-                                            >
-                                                {creditInfo.author.username}
-                                            </a> for the original project <a
-                                                href={`/preview/${creditInfo.id}`}
-                                            >
-                                                {creditInfo.title}
-                                            </a>.
-                                        </div>
+                                    <FlexRow className="action-buttons">
+                                        <Button className="action-button studio-button">
+                                            Add to Studio
+                                        </Button>
+                                        <Button className="action-button copy-link-button">
+                                            Copy Link
+                                        </Button>
+                                        {
+                                            sessionStatus === sessionActions.Status.FETCHED &&
+                                            Object.keys(user).length > 0 &&
+                                            user.id !== projectInfo.author.id && [
```

Comment:
Another candidate for a named show/hide variable.  Also with React 16 we can stop having these arrays, they added [`<React.Fragment>`](https://reactjs.org/docs/fragments.html) for this purpose! (I don't think our linter supports the shortened `<></>` syntax yet though)